packages/quicktype-core/src/language/CSharp.ts

```diff
@@ -301,6 +305,109 @@ function csNameStyle(original: string): string {
     );
 }
 
+function csNameStyleKeep(original: string): string {
+
+    const keywords = [
+        "abstract",
```

Comment:
Is this list declared somewhere else? I am sure we already have C# keywords defined somewhere, and we should not have two copies.

Reply:
> Is this list declared somewhere else? I am sure we already have C# keywords defined somewhere, and we should not have two copies.

I added it because I didn't find a list of keywords for C#, only for C++/Dart/Java and other languages